pkg/cache/application_info.go

```diff
@@ -52,9 +55,21 @@ type ApplicationInfo struct {
 	stateMachine      *fsm.FSM                   // application state machine
 	stateTimer        *time.Timer                // timer for state time
 
+	//TODO create a struct for it
+	statusChangeEventHandler commonevents.EventHandler
```

Comment:
change this to

```
eventHandlers handler.EventHandlers
```

just to keep consistency with `cluster_info`, `scheduler` etc

Reply:
done